mathesar/database/schemas.py

```diff
@@ -1,24 +1,28 @@
 from sqlalchemy.schema import CreateSchema
+from sqlalchemy import inspect
 
-from mathesar.database.base import engine, inspector
+from mathesar.database.base import create_mathesar_engine
 
+engine = create_mathesar_engine()
```

Comment:
Why not import the engine from `base.py`?

Reply:
I think eventually, we'll use the function for anything that's dealing with the engine to connect to the user table DB (as opposed to the Django table DB).  I was thinking that the exported engine (exported from `base.py`, that is) would be for the Django tables.  So, I'm planning to have that engine be created via a different function (the one that takes args), and have the one that makes the mathesar engine give the appropriate args for the user table database connection.

Reply:
I don't think we'll be using the engine for the Django tables at all, we'll just use the Django ORM.

Reply:
In that case, I think I'll leave it this way for the moment and reorganize when I split into separate DBs.  I do think the function version will be more flexible in the long haul, i.e., a given function call should know what connection it needs.  I suppose maybe it's premature while we only have one DB for user tables.